src/routes/Game.vue

```diff
@@ -152,15 +134,24 @@
         <div class="youtube_notice" v-if="srcMode === 'youtube'">
           Powered by YouTube.
           <br />
-          All copyright and revenue goes to the video owner.
+          Video copyright goes to the owner.
```

Comment:
```suggestion
          Video and music copyright goes to their respective copyright holder.
```

Also, do you happen to have any *copyright attribution* page and check if videos and music have suitable licenses that allow you to use them in this VSRG game by *fair use* clauses or explicit non-profit usage etc.? This is so that you don't get DMCA'd in any case 🤣

Reply:
I guess as all the plays are counted by youtube, and youtube provides the video, collects ad revenue and pays the copyright, I'm not really holding any copyrights here, I'm only embedding a video player you see ;)